health_vara_view.xml

```diff
@@ -34,6 +53,38 @@ contains the full copyright notices and license terms. -->
             <field name="act_window" ref="health_imaging.act_imaging_test_result_view"/>
         </record>
 
+        <!-- Improved icons for mammography imaging requests/results -->
+        <record model="ir.ui.icon" id="warning_icon">
+            <field name="name">gnuhealth.warning</field>
+            <field name="path">icons/warning.svg</field>
+        </record>
+
+        <record model="ir.ui.icon" id="imaging_icon">
+            <field name="name">gnuhealth.imaging-result</field>
+            <field name="path">icons/mammo_imaging_result.svg</field>
+        </record>
+
+        <record model="ir.ui.icon" id="new_imaging_request_icon">
+            <field name="name">gnuhealth-execute</field>
+            <field name="path">icons/mammo_new_imaging_request.svg</field>
+            <field name="sequence">5</field>
+        </record>
+
+        <record model="ir.ui.icon" id="imaging_request_icon">
+            <field name="name">gnuhealth.imaging-request</field>
+            <field name="path">icons/mammo_imaging_request.svg</field>
+        </record>
+
+        <menuitem parent="health_imaging.menu_imaging"
```

Comment:
Looks like we are creating new menu items for the custom `request` and `result` views. Are the original ones (including `draft`) still accessible somewhere in the menu tree?

Reply:
These menuitems existed already in gnuhealth, but I've restated them in order to change the icons.
These are in the left hand menu, and there was never a menuitem here for `draft` states.

It relates to the bottom two here:
![image](https://github.com/vara-ai/health_vara/assets/1367955/60ca4823-b2be-4976-9484-13fb3e8c3a44)

The new order icon was also replaced, but it had a unique icon, so I overwrote the icon, whereas the other two had shared icons, so I had to change the menuitems to point to distinct icons instead.